### PR TITLE
SYS-3133 - Adds workflow to check commit message and quantity of commits

### PR DESCRIPTION
**Pull request by felipem1210**

### CODE REVIEW

**Review by agurebora — DISMISSED** [review]

File: .github/workflows/check-commit.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+name: commit message check
```
Comment (mattialacroce) [inline .github/workflows/check-commit.yml:1]: The name here is not generic for both the tasks the job will perform, it seems only the commit message will be checked, which is not true (commit number is validated as well). How does "branch commit validation" sound here?

Comment (felipem1210) [reply .github/workflows/check-commit.yml:1]: Commits check, Like it

Comment (mattialacroce) [reply .github/workflows/check-commit.yml:1]: commit checks! [](edited-with-Axolo)

Comment (mattialacroce) [reply .github/workflows/check-commit.yml:1]: okkk [](edited-with-Axolo)

**Review by mattialacroce — APPROVED** [review]

File: .github/workflows/check-commit.yml (hunk 1 above)
Comment (felipem1210) [reply .github/workflows/check-commit.yml]: done [](edited-with-Axolo)

**Pull request merged**